source/creating_maps_charts/02_creating_advanced_charts/analyze.rst

```diff
@@ -1,41 +1,40 @@
 Analyzing with the Chart Builder
 ================================
 
-.. admonition:: Prerequisite
-   :class: important
+The Chart Builder tool allows you to visualize multiple datasets on a single customized chart.
 
-   Read how to :doc:`configure visualizations </publishing_data/07_configuring_visualizations/03_configuring_analyze_view/analyze>` for a dataset beforehand.
+It is usually made available by the domain administrator via a **Charts** link in the header menu. You can also access the Chart Builder through the following URL: ``http://<DOMAIN ADDRESS>/chart``.
 
-With the Chart Builder tool, you can visualize multiple datasets on a single customized chart. You can save your chart and easily share it, embed it, or use it as a widget.
+The Chart Builder is more flexible than **Analyze** tab. With the Chart Builder, you're not limited to one dataset. You can combine combining different sources from your domain and the Opendatasoft network.
```

Comment:
**The typo fixer**

_Insert suspenseful music here_

- Combine says : "This documentation is too small for the both of us, gringo.
- Combining answers : "Well, I guess one of us is about to be deleted then ..."

One of them gets deleted, the typo is fixed.
The end